docs/queriying_documents.md

```diff
@@ -248,12 +248,20 @@ This is the equivalent of a MongoDB `updateOne`.
 You can also do bulk updates like this:
 
 ```python
-movies = await Movie.query({Movie.year: 1970}).update({Movie.year: 1980})
+movies = await Movie.query({Movie.year: 1970}).update(year=1980)
 ```
 
 Here we do an update to change the `year` of all 1970 movies to 1980.
 
-The returned result is a list of update `Movie` instances.
+Now let's update multiple fields in a bulk update:
+
+```python
+movies = await Movie.query({Movie.name: "Example"}).update(year=1970, name="Another Movie")
+```
+
```

Comment:
Honestly I'd rather use the above example to update multiple fields, I think that's minimum documentation and will give the idea. What do you think?

Reply:
yes, I mean update the existing example to show that multiple fields is also possible.

Reply:
Ok :+1: